app/routes/events/EventAbacardRoute.ts

```diff
@@ -23,7 +23,7 @@ const loadData = async (props, dispatch): any => {
 };
 
 const mapStateToProps = (state, props) => {
-  const query = qs.parse(props.location.search);
+  const query = qs.parse(props.location.search).q;
```

Comment:
what this do

Reply:
It's the quary parameter for the search, eg. /abacard?q=Ludvig

This is an unrelated bug also fixed here with the search results not clearling when selecting a user